new-log-viewer/package.json

```diff
@@ -5,10 +5,13 @@
   "main": "src/index.tsx",
   "scripts": {
     "build": "webpack --config webpack.prod.js",
+    "start": "webpack serve --open --config webpack.dev.js",
+
     "lint": "npm run lint:check",
-    "lint:check": "eslint src webpack.*.js --max-warnings 0",
-    "lint:fix": "npm run lint:check -- --fix",
-    "start": "webpack serve --open --config webpack.dev.js"
+    "lint:check": "npm-run-all --sequential --continue-on-error lint:check:*",
+    "lint:check:css": "stylelint src/**/*.css --formatter github",
```

Comment:
Sure. I think we can add the problem matcher for the `string` formatter in another PR.

Reply:
Before I make the changes let's take a look at the approaches we can take:
1. Use a custom formatter in Stylelint and a custom action in the GitHub workflow.
   PROS: We can view the errors in a nicely formatted way and we can have the errors annotated inline by GitHub.
   CONS: setting up custom stuff takes time; we might not want to use custom stuff given they might not always be compatible with the tooling upgrades / GitHub workflow, which means we might not want to maintain it as a long term solution.
2. Use the default `string` formatter.
   PROS: It's the default formatter which means it should have the best support from the tooling contributors.
   CONS: We lose the ability to have the errors annotated inline (until we move the files into the project root). It outputs relative paths, but GitHub workflow problem matchers only resolves paths from the project root. Supposedly we can check in the problem matcher, which currently does nothing, in the same PR to save another PR, but it's hard to verify if things would actually work. If we spot any issues after we move the log-viewer files, we might still need to fix in the PR we move the files or another PR.
3. Use the `github` formatter.
   PROS: We don't need any custom plugin to get the GitHub annotation working.
   CONS: We will see a deprecated warning in the workflow results. We're using a deprecated but not yet removed formatter. If we don't upgrade the Stylelint version the formatter should still be there, and it's less likely we would upgrade it before we move the new-log-viewer files and migrate to the problem matcher approach. 

I still think Approach 3 gives us the most benefits, but all CONs are minor since we might always need another PR to improve the lint experience after we move the new-log-viewer files.